pkg/kubectl-argo-rollouts/cmd/create/create.go

```diff
@@ -137,6 +137,17 @@ func unmarshal(fileBytes []byte, obj interface{}) error {
 	}
 }
 
+func (c *CreateOptions) getNamespace(un unstructured.Unstructured) string {
+	ns := c.ArgoRolloutsOptions.Namespace()
+	if md, ok := un.Object["metadata"]; ok {
+		metadata := md.(map[string]interface{})
```

Comment:
may panic here if metadata is not a map in the resource file.

Reply:
not write matadata name is invalid resource file,
it's error